9.1.9 Security release notes (#4326)
elastic/docs-content: release-notes/elastic-security/index.md

* Fixes an issue in {{elastic-defend}} that could result in a crash if a specified {{ls}} output configuration contained a certificate that couldn't be parsed.
* Fixes CVE-2025-37735 ([ESA-2025-23](https://discuss.elastic.co/t/elastic-defend-8-19-6-9-1-6-and-9-2-0-security-update-esa-2025-23/383272)) in {{elastic-defend}} on Windows which could allow a low-privilege attacker to delete arbitrary files on the system and potentially escalate privileges to SYSTEM. Windows 11 24H2 includes changes which make this issue harder to exploit.

## 9.1.9 [elastic-security-9.1.9-release-notes]

### Features and enhancements [elastic-security-9.1.9-features-enhancements]
* Shows session view in full height [#245888]({{kib-pull}}245888).
* Shows analyzer in full height [#245857]({{kib-pull}}245857).
* Hardens {elastic defend} against cloud filter rebinding.
* Improves general system responsiveness while {elastic-defend} is installed.

### Fixes [elastic-security-9.1.9-fixes]
* Standardizes how to log errors [#245030]({{kib-pull}}245030).
* Fixes an issue that could prevent {elastic-defend} from properly handling upgrades when Tamper Protection is enabled.

> * {elastic-defend} will no longer reload its policy if nothing has functionally changed from the previous policy.

Reviewer: ```suggestion
* {elastic-defend} no longer reloads its policy if nothing has functionally changed from the previous policy.
```

* Fixes an issue in {{elastic-defend}} that could result in a crash if a specified {{ls}} output configuration contained a certificate that couldn't be parsed.
* Fixes CVE-2025-37735 ([ESA-2025-23](https://discuss.elastic.co/t/elastic-defend-8-19-6-9-1-6-and-9-2-0-security-update-esa-2025-23/383272)) in {{elastic-defend}} on Windows which could allow a low-privilege attacker to delete arbitrary files on the system and potentially escalate privileges to SYSTEM. Windows 11 24H2 includes changes which make this issue harder to exploit.

## 9.1.9 [elastic-security-9.1.9-release-notes]

### Features and enhancements [elastic-security-9.1.9-features-enhancements]
* Shows session view in full height [#245888]({{kib-pull}}245888).
* Shows analyzer in full height [#245857]({{kib-pull}}245857).
* Adds for Bind Link requests mitigation for {elastic defend}.
* Improves general system responsiveness while {elastic-defend} is installed.

> * Uses mandatory template {elastic-defend} which will be substituted with proper brand name, which is {elastic-defend}.

Reviewer: That's a leftover from our PR template.  The engineer forgot to delete it.

```suggestion
```

> * Reduces the number of I/O operations performed by {elastic-defend} for file event enrichment. This reduction is more pronounced when Ransomware Protection is not in use.

Reviewer: Unfortunately this didn't land in time.

```suggestion
```

* Fixes an issue in {{elastic-defend}} that could result in a crash if a specified {{ls}} output configuration contained a certificate that couldn't be parsed.
* Fixes CVE-2025-37735 ([ESA-2025-23](https://discuss.elastic.co/t/elastic-defend-8-19-6-9-1-6-and-9-2-0-security-update-esa-2025-23/383272)) in {{elastic-defend}} on Windows which could allow a low-privilege attacker to delete arbitrary files on the system and potentially escalate privileges to SYSTEM. Windows 11 24H2 includes changes which make this issue harder to exploit.

## 9.1.9 [elastic-security-9.1.9-release-notes]

### Features and enhancements [elastic-security-9.1.9-features-enhancements]
* Shows session view in full height [#245888]({{kib-pull}}245888).
* Shows analyzer in full height [#245857]({{kib-pull}}245857).

> * Adds for Bind Link requests mitigation for {elastic defend}.

Author: @gabriellandau would this work instead:

```suggestion
* Hardens {elastic defend}  against cloud filter rebinding.
```

> * Adds for Bind Link requests mitigation for {elastic defend}.

Reviewer: Yes, minus a space :)

```suggestion
* Hardens {elastic defend} against cloud filter rebinding.
```

* Improves general system responsiveness while {elastic-defend} is installed.

> * Uses mandatory template {elastic-defend} which will be substituted with proper brand name, which is {elastic-defend}.

Reviewer 2: I'm also a bit unclear on what this means, an edit from a technical reviewer would be great
